cmd/incusd/main_forknet.go

```diff
@@ -135,7 +135,11 @@ static void forkdonetdhcp() {
 	}
 
 	// Set the process title.
-	(void)setproctitle("[incus DHCP] eth0");
+	char *workdir = advance_arg(false);
+	char *title = malloc(sizeof(char)*strlen(workdir)+18);
+	sprintf(title, "[incus dhcp] %s eth0", workdir);
+
+	(void)setproctitle(title);
```

Comment:
Please handle the malloc for title failing.  It'll crash setproctitle if title is NULL.